### ISSUE TITLE
Support for NamedConstructor

### ISSUE THREAD

**Issue by deian** [tabatkins/bikeshed#516]
Bikeshed currently does not have support for named constructors. COWL is currently using static methods to bypass this, but it would be great if we didn't have to. While digging around, I believe there were a few other specs that were doing the same.

Thanks! 
Ref: https://w3c.github.io/webappsec-cowl/#issue-56e73af5

**Comment by tabatkins** [comment]
Yup, plan to support it, there just wasn't anyone using it, so I hadn't spent time on it. I'll prioritize this. ^_^

**Comment by deian** [comment]
Thanks! :smile:
